kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -362,4 +366,21 @@ public void deleteAllowed(Subscriber subscriber) {
             subscriptionService.deletePreconditionCheck(subscription);
         }
     }
+
+    @Override
+    public void deactivateAllSubscriptionsForSubscriber(long callingNumber) {
+        LOGGER.info("Recieved Release Number {} for Deactivation.", callingNumber);
+        Subscriber subscriberByMsisdn = this.getSubscriber(callingNumber);
```

Comment:
Add a log to  check total number of subscription found and counter to validate each and every subscription got deactivated.